doc/transform.ipynb

```diff
@@ -2335,8 +2335,14 @@
    "cell_type": "markdown",
```

Comment:
<p>Maybe we should even take a step back and call it "Interpolate to different depth coordinates"?</p><p> </p><p>Reading this, I think that this example is great, but very complex. It involves a lot of other dependencies, that might distract the reader from the grid.transform part.</p><p> </p><p>My suggestion:</p><ul><li>Keep this notebook separately, and lets turn it into an example in <a href="https://github.com/xgcm/xgcm-examples" target="_blank">xgcm-examples</a>. To me that is a place to show of 'full stack' examples.</li></ul><p> </p><ul><li>For this part here, I would just take the dataset that we already use for the other examples (GODAS) and convert it to a uniform depth (you can simply specify that by <code>np.linspace</code> or <code>np.arange</code>). The point to make here is that we can use <code>grid.transform</code> so simply interpolate on different depth values.</li></ul><p> </p><p>What do you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgcm/xgcm/pull/322/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<blockquote>Keep this notebook separately, and lets turn it into an example in&nbsp;<a href="https://github.com/xgcm/xgcm-examples" target="_blank">xgcm-examples</a>. To me that is a place to show of 'full stack' examples.</blockquote><p>	Sounds good!</p><p> </p><blockquote>For this part here, I would just take the dataset that we already use for the other examples (GODAS) and convert it to a uniform depth (you can simply specify that by&nbsp;<code>np.linspace</code>&nbsp;or&nbsp;<code>np.arange</code>). The point to make here is that we can use&nbsp;<code>grid.transform</code>&nbsp;so simply interpolate on different depth values.</blockquote><p> </p><p>I can work on an example, but I wanted to confirm that this is a correct way of assigning values:</p><p> </p><pre class="ql-syntax" spellcheck="false">grid = Grid(cmip6_thetao, coords={'Z': {'center': 'lev', 'outer': 'lev_bounds'},
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;'X': {'center': 'lon'},
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;'Y': {'center': 'lat'}},
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;periodic=False
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;)
</pre><p> </p><p> </p><p> </p>